src/components/itemMediaInfo/itemMediaInfo.js

```diff
@@ -71,8 +72,9 @@ const attributeDelimiterHtml = layoutManager.tv ? '' : '<span class="hide">: </s
             const size = `${(version.Size / (1024 * 1024)).toFixed(0)} MB`;
             html += `${createAttribute(globalize.translate('MediaInfoSize'), size)}<br/>`;
         }
-        for (let i = 0, length = version.MediaStreams.length; i < length; i++) {
-            const stream = version.MediaStreams[i];
+        const streams = version.MediaStreams.sort((a, b) => itemHelper.sortTracks(a, b));
```

Comment:
Couldn't you simplify these to just `version.MediaStreams.sort(itemHelper.sortTracks)`?